src/types/types.ts

```diff
@@ -600,6 +600,15 @@ export interface SpamInfo {
   classifications: NftSpamClassification[];
 }
 
+/** Block time and number at which an NFT was acquired. */
```

Comment:
Add a comment that this is only available when specifying the orderby = transfertime.